src/java/fr/paris/lutece/portal/service/editor/ParserBbcodeServlet.java

```diff
@@ -72,19 +73,21 @@ public ParserBbcodeServlet( )
      * 
      * @param request  servlet request
      * @param response servlet response
-     * @throws ServletException the servlet Exception
-     * @throws IOException      the io exception
      */
     protected void processRequest( HttpServletRequest request, HttpServletResponse response )
-            throws ServletException, IOException
     {
         String strValue = request.getParameter( PARAMETER_DATA );
         String strEscaped = StringEscapeUtils.escapeHtml( strValue );
         String strValueReturn = ( strValue != null ) ? EditorBbcodeService.getInstance( ).parse( strEscaped ) : "";
-        OutputStream out = response.getOutputStream( );
-        out.write( strValueReturn.getBytes( AppPropertiesService.getProperty( PROPERTY_ENCODING ) ) );
-        out.flush( );
-        out.close( );
+        
+        try ( OutputStream out = response.getOutputStream( ) )
+        {
+            out.write( strValueReturn.getBytes( AppPropertiesService.getProperty( PROPERTY_ENCODING ) ) );
+        }
+        catch ( IOException e )

```

Comment:
Isn't it better for the servlet's client to respond with a proper HTTP 500 error rather than silently failing ?